hotel-management/src/components/Dropdown/index.css

```diff
@@ -0,0 +1,27 @@
+.dropdown {
+  position: absolute;
+  left: 50%;
+  top: 100%;
+  z-index: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  transform: translate(-50%, 10px);
```

Comment:
Why need to transform, u already declare position by using left and top

Reply:
First, I need to `translateX(-50%)` so the dropdown will be 100% centered, and with `top: 100%` the dropdown will align to bottom edge of the parent then `translateY(10x)` to easily make gapping

Reply:
Which element is this dropdown centered according to? What if parent element has no position property? What if I don't want center the dropdown? Or u mean center the dropdown item instead the whole dropdown?